input/definitions/Resource/StructureDefinition-cqf-doNotPerformReason.json

```diff
@@ -0,0 +1,91 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "cqf-doNotPerformReason",
+  "text": {
+    "status": "extensions"
+  },
+  "url": "http://hl7.org/fhir/StructureDefinition/cqf-doNotPerformReason",
+  "version": "5.0.0",
+  "name": "CQFDoNotPerformReason",
+  "title": "CQF Do Not Perform Reason",
+  "status": "active",
+  "experimental": false,
+  "date": "2024-05-20",
+  "publisher": "HL7 International / FHIR Infrastructure",
+  "contact": [
+    {
+      "name": "Clinical Quality Information WG",
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://www.hl7.org/Special/committees/fiwg"
+        }
+      ]
+    }
+  ],
+  "description": "Indicates the reason the event was not performed.",
+  "jurisdiction": [
+    {
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166",
+          "code": "US"
+        }
+      ]
+    }
+  ],
+  "fhirVersion": "4.0.1",
+  "mapping": [
+    {
+      "identity": "rim",
+      "uri": "http://hl7.org/v3",
+      "name": "RIM Mapping"
+    }
+  ],
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "StructureDefinition"
```

Comment:
This should not be StructureDefinition, this should be limited to those resources we used it on in QICore profiles:
DeviceRequest and ServiceRequest